### PR TITLE
Change data & config dir of parsec to remove the suffix `-alpha`

### PR DESCRIPTION
**Pull request by FirelightFlagboy**

### CODE REVIEW

File: cli/src/tests.rs [hunk 1]
```diff
@@ -875,7 +875,7 @@ async fn remove_device(tmp_path: TmpPath) {
 
     let alice_device_file = tmp_path
         .join("config")
-        .join("parsec-v3-alpha")
+        .join("parsec3/libparsec")
```
Comment (Max-7) [inline cli/src/tests.rs:878]: Probably doesn't change much but you could also separate those:
```
.join("config")
.join("parsec3")
.join("libparsec")
...
```

File: libparsec/crates/platform_device_loader/src/lib.rs [hunk 2]
```diff
@@ -26,6 +26,65 @@ pub const ARGON2ID_DEFAULT_PARALLELISM: u32 = 1;
...
+    #[cfg(target_arch = "wasm32")]
+    {
+        PathBuf::from("")
+    }
+    #[cfg(not(target_arch = "wasm32"))]
+    {
+        let mut path = if let Ok(config_dir) = std::env::var(PARSEC_CONFIG_DIR) {
+            PathBuf::from(config_dir)
+        } else {
+            // TODO: check MacOS path correspond to Parsec v2
```
Comment (Max-7) [inline libparsec/crates/platform_device_loader/src/lib.rs:61]: Is this comment still relevant?

**Review by Max-7 — APPROVED** [review]

File: libparsec/crates/platform_device_loader/src/lib.rs (hunk 2 above)
Comment (FirelightFlagboy) [reply libparsec/crates/platform_device_loader/src/lib.rs:61]: nop

File: cli/src/tests.rs (hunk 1 above)
Comment (FirelightFlagboy) [reply cli/src/tests.rs]: Just do everything on one `join` expect for that last value which is dynamic

**Pull request merged**